src/scrabble.js

```diff
@@ -1,5 +1,51 @@
-function scrabble() {
-
+const lettervalue = {
```

Comment:
I like that you use an object to hold all the values for the letters, this makes statements a lot more readable and the code generally more flexible.